framework/util/hash.h

```diff
@@ -98,6 +98,117 @@ std::size_t hash_range(It first, It last)
     return seed;
 }
 
+//! stripped-out finalizer from murmur3_32
+inline uint32_t murmur3_fmix32(uint32_t h)
```

Comment:
Do we need to add copyright notice info in our header if we're pulling source from elsewhere?

Reply:
I'm not entirely sure, the algorithm itself should be public-domain. 
the code itself was written by me, based on an older implementation I already had and then c++-templatized.
so there was not really a source I could point to, other then the original author's repo
-> https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp

the license-header says:
_// MurmurHash3 was written by Austin Appleby, and is placed in the public
// domain. The author hereby disclaims copyright to this source code._

should we adopt that?

Reply:
thanks @MarkY-LunarG!